example/dash_analytics_example.dart

```diff
@@ -40,13 +38,19 @@ final Analytics analytics = Analytics(
   branch: 'ey-test-branch',
   flutterVersion: 'Flutter 3.6.0-7.0.pre.47',
   dartVersion: 'Dart 2.19.0',
+  platform: 'macos',
 );
 
 void main() async {
   DateTime start = DateTime.now();
   print('###### START ###### $start');
 
   print(analytics.telemetryEnabled);
+  analytics.sendEvent(
+    eventName: 'testing_from_dash',
```

Comment:
Look into Dart's enhanced enums for keeping event names

Reply:
https://dart.dev/guides/language/language-tour#enumerated-types

Reply:
Thank you! This is perfect, snippet of sample code

```dart
enum Events {
  // List all of the events here
  runTime('run_time'),
  hotReloadTime('hot_reload_time');

  // The positional arg for the default constructor are the labels we have
  // defined in the values above
  const Events(this.label);
  final String label;
}

void main() {
  const event = Events.runTime;
  print(event.label); // Returns 'run_time'
}
```